tests/simple/common/shell_tests.sh

```diff
@@ -0,0 +1,13 @@
+test_shell_syntax() {
+	bash -n "${1}"
+}
+
+test_shellcheck() {
+	if ! which shellcheck ; then
+		echo "ShellCheck not found: skipping..."
+		return 0
+	fi
+
+	shellcheck -x -s bash -e SC2181,SC2029,SC1091,SC1090 "${1}"
```

Comment:
This sneaks in additional exceptions, which should get into their own conscious commit, imho.